### PR TITLE
21.05

### PR DESCRIPTION
**Pull request by t184256**
@Gerschtli, got time for a review? Nothing fancy going on in here.

Also, got any more suggestions for the changelog?

### CODE REVIEW

**Comment by Gerschtli** [comment]
Diff LGTM, didn't grep for 20.09 by myself but looks like you found every occurence.

Did not test it on my device, will check it on the weekend.

**Comment by Gerschtli** [comment]
Please do not forget to push all packages and the ones affected by overlays to push to the binary cache :)

File: modules/workaround-make.nix [hunk 1]
```diff
@@ -59,6 +59,11 @@ in
   ###### implementation
 
   config = {
+    warnings = lib.optionals cfg.enable [
+      ("system.workaround.make-posix-spawn.enable " +
+       "should no longer be needed on 21.05")
```
Comment (Gerschtli) [inline modules/workaround-make.nix:64]: Maybe clarify that this option will be removed in 21.11 and if anyone still needs this, they should create an issue?

**Review by Gerschtli — APPROVED** [review]
Tested it and could not find any issues :)

File: modules/home-manager.nix [hunk 2]
```diff
@@ -72,7 +72,7 @@ in
         installation of user packages through the
         <option>environment.packages</option> option.
       '' // {
-        default = versionAtLeast config.system.stateVersion "20.09";
+        default = versionAtLeast config.system.stateVersion "21.05";
       };
```
Comment (Gerschtli) [inline modules/home-manager.nix:76]: This should not be changed!

Comment (t184256) [reply modules/home-manager.nix:76]: uh-oh, fixup time

File: modules/workaround-make.nix (hunk 1 above)
Comment (t184256) [reply modules/workaround-make.nix]: expanded the warning text

File: modules/home-manager.nix (hunk 2 above)
Comment (t184256) [reply modules/home-manager.nix]: changed back, thanks for catching it!

**Comment by t184256** [comment]
OK, let's release. Thank you for your review!

**Pull request merged**